src/components/billingRecord/IFXBillingRecordList.vue

```diff
@@ -181,16 +186,14 @@ export default {
       return message
     },
     async getFullBillingRecordByItemIndex(index) {
-      let br = this.items?.[index]
-      if (br?.billingRecordStates?.length) {
-        // Full record is already there
+      let br = this.items[index]
+      if (br.billingRecordStates?.length) {
         return br
       }
       if (br.id) {
         // Go get it
         br = await this.apiRef.getByID(this.facility.invoicePrefix, br.id)
         this.$set(this.items, index, br)
-        console.log('the items now ', this.items)
         return br
       }
       console.log(`Billing record with id not found at item index ${index}`)
```

Comment:
I assume this shouldn't happen but you want to log them somewhere?

Reply:
I think it's a developer only thing.